Pages/Index.cshtml.cs

```diff
@@ -17,30 +17,24 @@ public class IndexModel : PageModel
     {
         public string GetData(string endpoint)
         {
-            string downloadedData = "";
             using (WebClient webClient = new WebClient())
             {
-                downloadedData = webClient.DownloadString(endpoint);
+                return webClient.DownloadString(endpoint);
             }
-            return downloadedData;
         }
 
         public void OnGet()
         {
-            using (var WebClient = new WebClient())
-            {
                 
-                string jsonstring = GetData("https://data.cityofchicago.org/resource/xq83-jr8c.json");
```

Comment:
Removed redundant code

Reply:
Good to remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.